assets/src/components/admin/admin_tables.tsx

```diff
@@ -451,6 +451,16 @@ const BuswayV2ScreensTable = (): JSX.Element => {
   return <AdminTable columns={columns} dataFilter={dataFilter} />;
 };
 
+const OnBusScreensTable = (): JSX.Element => {
+  const dataFilter = ({ app_id }) => {
+    return app_id === "on_bus_v2";
+  };
+
+  const columns = [...v2Columns];
```

Comment:
I think we might now need to split out "Header" from `v2Columns`, since this is resulting in on-bus screens being configurable with a header, but they don't actually have such a field.

Reply:
this is a good point! Moved header out into a separate column and updated the other tables accordingly.